vms/platformvm/service.go

```diff
@@ -511,6 +511,83 @@ func (s *Service) GetUTXOs(_ *http.Request, args *api.GetUTXOsArgs, response *ap
 	return nil
 }
 
+/*
+ *****************************************************
+ ******************* Get Subnet **********************
+ *****************************************************
+ */
+
+// GetSubnetArgs are the arguments to GetSubnet
+type GetSubnetArgs struct {
+	// ID of the subnet to retrieve information about
+	SubnetID ids.ID `json:"subnetID"`
+}
+
+// GetSubnetResponse is the response from calling GetSubnet
+type GetSubnetResponse struct {
+	// ID of the subnet
+	SubnetID ids.ID `json:"subnetID"`
```

Comment:
Why do we return the subnetID in the response? Won't the caller already know it from the request?

Reply:
Yup. I had some confusion when making the rpc call with incorrect field name in the json (my mistake). The json was validated and the call returned info for platform without notice on my side. So I added the subnet ID. But it is best to just remove this and I also like returning error when requesting for platform (it is a way to validate the json param is correct).